backend/app/tests/utils/test_data.py

```
@@ -0,0 +1,116 @@
+from typing import List
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)